framework/tests/flowexpression/ValueLiterals.java

```diff
@@ -0,0 +1,11 @@
+import testlib.flowexpression.qual.FlowExp;
```

Comment:
This looks like it shouldn't be showing up as a diff in your pull request, either.